bindings/js/test2.js

```diff
@@ -1,29 +1,32 @@
 // jshint esversion: 8
 
+// We demonstrate how to write a sample program that uses the high-level HACL*
+// API. Run this with `node test2.js`. See `test.html` for a version to be run
+// directly from within the browser.
+
 var HaclWasm = require('./api.js');
 var loader = require('./loader.js');
 
-function buf2hex(buffer) {
-  return Array.prototype.map.call(new Uint8Array(buffer), function(x) {
-    return ('00' + x.toString(16)).slice(-2);
-  }).join('');
-}
+// Test helpers
+// ------------
+
+const buf2hex = buffer => [...new Uint8Array(buffer)].map(x => `00${x.toString(16)}`.slice(-2)).join('');
```

Comment:
Shouldn't this be provided by the Kremlin modules instead of re-implementing it in each test ?

Reply:
yes, I'll do that in the next PR